### PR TITLE
[ENH] Add Level objects to channels.json for motion

### PR DESCRIPTION
**Pull request by effigies**
- Replaces #1524.

This was initially comments on #1524, but it became a lot and easier to just make a proposal. Making pull requests against other people's branches often fails, so might as well just make an alternative PR.

The things worth noting:

1) This is essentially taking #1573 as accepted. I'm fine with it, but this is a change from the current spec, where these fields MUST be strings. This probably needs a PR on its own. 
image
2) I think adding `"n/a"` is strictly worse than omitting a field. In one case I can test for presence.
3) "ReferenceFrameDescription" is redundant when it is in `sidecar.reference_frame.Levels.<rf_name>` object. I just changed it to Description.

### CODE REVIEW

**Comment by sjeung** [comment]
Thank you, I agree with all of the proposed changes. 

Example data sets in [this PR](https://github.com/bids-standard/bids-examples/pull/418) reflects them already (renamed ReferenceFrameDescription -> Description). 
Once Julius approves my change to spec [this PR](https://github.com/bids-standard/bids-specification/pull/1524) will be up to date with it too. (Removed unnecessary n/a options and renamed ReferenceFrameDescription -> Description)


> 1. This is essentially taking [allow Levels values (in sidecar for TSV files) to be objects #1573](https://github.com/bids-standard/bids-specification/issues/1573) as accepted. I'm fine with it, but this is a change from the current spec, where these fields MUST be strings. This probably needs a PR on its own.

Do you plan to open this one or should I have a look into implementing this?

**Comment by effigies** [comment]
> Do you plan to open this one or should I have a look into implementing this?

That was done in #1603.

File: src/schema/objects/columns.yaml [hunk 1]
```diff
@@ -377,6 +377,18 @@ reference__ieeg:
     - type: string
       enum:
         - n/a
+reference_frame:
+  name: reference_frame
+  display_name: Reference frame
+  description: |
+    Specification of a reference frame in which the motion data are to be interpreted.
+    The description of the levels in `*_channels.json` SHOULD use `RotationRule`,
+    `RotationOrder` and `SpatialAxis` for the specification.
```
Comment (sappelhoff) [inline src/schema/objects/columns.yaml:386]:
```suggestion
    `RotationOrder`, and `SpatialAxis`, and MAY use `Description` for the specification.
```

**Review by sappelhoff — APPROVED** [review]
I think mentioning `Description` is a good idea, but feel free to ignore if you disagree. Anyhow, +1 to merge

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**